npm/webpack-dev-server/src/aut-runner.ts

```diff
@@ -21,11 +29,22 @@ export function init (importPromises, parent = (window.opener || window.parent))
   Cypress.onSpecWindow(window, importPromises)
   Cypress.action('app:window:before:load', window)
 
+  // In this variable, we save head
+  // innerHTML to account for loader installed styles
+  let headInnerHTML = ''
+
+  // before the run starts save
+  Cypress.on('run:start', () => {
+    headInnerHTML = document.head.innerHTML
+  })
+
   // Before all tests we are mounting the root element, **not beforeEach**
   // Cleaning up platform between tests is the responsibility of the specific adapter
   // because unmounting react/vue component should be done using specific framework API
   // (for devtools and to get rid of global event listeners from previous tests.)
   Cypress.on('test:before:run', () => {
+    document.head.innerHTML = headInnerHTML
```

Comment:
Sorry, @elevatebart  I appreciate your help, but I still don't understand why this should be a part of webpack-dev-server. It looks like not the job of the dev server. 

If somebody is going to not use our framework – let him cleanup his code by his own. It just looks like trying to solve a problem that is not exists.

For reference: testing library provides auto cleanup just like we do, but if you do can also use it manually I think for tests like this 

https://testing-library.com/docs/svelte-testing-library/api/#cleanup